sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -482,12 +482,20 @@ manifest({
             hex(CONST.BFV.BOOT_POLICY.BAD_IDENTIFIER),
             hex(lc_state_val),
         ),
+        otp = ":otp_img_shutdown_output_{}".format(lc_state.lower()),
```

Comment:
isn't otp is already part of the bitstream? Is there a reason for specifying it here as well?

Reply:
currently, the `cw310_params` specify an `otp` parameter, and it defaults to the base RMA image. So before this modification, the `sh_test` rule that gets instantiated by this `opentitan_functest` depended on the custom OTP image that was a dep of the bitstream, and the RMA base OTP image, which wasn't quite right (i.e. both images were getting generated). We should probably just remove this `otp` parameter from the `cw310_params` since it is a bitstream dep (we will need to keep it though for `dv_params` and `verilator_params`.